source/_components/doorbird.markdown

```diff
@@ -74,56 +75,62 @@ devices:
       description: If your DoorBird cannot connect to the machine running Home Assistant because you are using dynamic DNS or some other HTTP configuration (such as HTTPS), specify the LAN IP of the machine here to force a LAN connection.
       required: false
       type: string
-    monitored_conditions:
-      description: Monitor motion and/or doorbell events for this device.
+    events:
+      description: Custom event names to be registered on the device.  User defined values.  Special characters should be avoided.
       required: false
-      type: string
-      keys:
-        doorbell:
-          description: Monitor doorbell events.
-        motion:
-          description: Monitor motion events (Motion monitoring must be enabled on the doorstation via DoorBird app).
-        relay:
-          description: Monitor relay events. This event is fired even if a relay is not physically connected to the door station.  Can be used to lock/unlock any smart lock present in Home Assistant via the Doorbird app.
-
+      type: list
+      
 {% endconfiguration %}
 
 ## {% linkable_title Full example %}
 
 ```yaml
 doorbird:
-  token: YOUR_DOORBIRD_TOKEN
   devices:
     - host: DOORBIRD_IP_OR_HOSTNAME
       username: YOUR_USERNAME
       password: YOUR_PASSWORD
+      token: CUSTOM_TOKEN_1
       hass_url_override: HASS_URL
       name: Front Door
     - host: DOORBIRD_IP_OR_HOSTNAME
       username: YOUR_USERNAME
       password: YOUR_PASSWORD
+      token: CUSTOM_TOKEN_2
       name: Driveway Gate
       monitored_conditions:
```

Comment:
Should we change this?

Reply:
Previously, it was one level higher in the config structure, so all devices used the same token value. Now, the token is used to identify devices when calling the reset URL to clear the registered events. What kind of change do you propose?

Reply:
I was thinking about `monitored_conditions`. I thought we had removed that option.

Reply:
Ah... totally overlooked that.